### PR TITLE
fix stability test, add `v1` to statefulset apiVersions

### PR DESCRIPTION
**Pull request by weekface**
### Check List 

Tests 

 - Stability test

Code changes

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:
 
 ```release-note
NONE
 ```

### CODE REVIEW

File: pkg/webhook/statefulset/statefulset.go [hunk 1]
```diff
@@ -68,22 +70,20 @@ func AdmitStatefulSets(ar v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
...
 
-	raw := ar.Request.OldObject.Raw
-	set := apps.StatefulSet{}
-	if _, _, err := deserializer.Decode(raw, nil, &set); err != nil {
-		err := fmt.Errorf("statefulset %s/%s, decode request failed, err: %v", namespace, name, err)
-		glog.Errorf(err.Error())
+	stsObjectMeta, stsPartition, err := getStsAttributes(ar.Request.OldObject.Raw, apiVersion)
+	if err != nil {
+		glog.Errorf("statefulset %s/%s, decode request failed, err: %v", namespace, name, err)
 		return util.ARFail(err)
```
Comment (onlymellb) [inline pkg/webhook/statefulset/statefulset.go]: I think this return error is not meaningful, should return a more detailed error, just like the error output above.

File: tests/actions.go [hunk 2]
```diff
@@ -447,7 +447,7 @@ func (oa *operatorActions) DeployOperator(info *OperatorConfig) error {
 	}
 
 	// deploy statefulset webhook and configuration to hijack update statefulset opeartion
-	cmd = fmt.Sprintf("kubectl apply -f %s/webhook.yaml", oa.manifestPath(info.Tag))
+	cmd = fmt.Sprintf(`sed 's/apiVersions: \["v1beta1"\]/apiVersions: ["v1", "v1beta1"]/' %s/webhook.yaml | kubectl apply -f -`, oa.manifestPath(info.Tag))
```
Comment (onlymellb) [inline tests/actions.go]: `manifests/webhook.yaml` has been modified, so can you omit this operation?

Comment (weekface) [reply tests/actions.go]: no, `v1.0.1` or prior only has one `v1beta1`. We must change them too in the stability test case.

File: pkg/webhook/statefulset/statefulset.go (hunk 1 above)
Comment (weekface) [reply pkg/webhook/statefulset/statefulset.go]: Done

**Pull request merged**